Example/RSFormViewExampleApp/SignupExampleViewModel.swift

```diff
@@ -88,6 +88,8 @@ class SignupExampleViewModel: FormViewModel {
   
   func addressHeaderItem() -> FormItem {
     let item = FormItem()
+    item.contraintsConfigurator.headerLabelTopMargin = CGFloat(20)
+    item.contraintsConfigurator.headerLabelBottomMargin = CGFloat(80)
```

Comment:
since this is a header item for the address below it makes more sense for the text to be closer to the related fields. Lets think of adding a hint text to another field (smaller font and close to the field above)

Reply:
The numbers where exaggerated to see if the configurations where in fact being set, will change that so it doesn't look so ugly 🥴. We can consider adding an optional hint text though, good idea!